src/documentation/docs/components/NewsDetail.mdx

```diff
@@ -13,47 +15,92 @@ import PropsTableRow from "@/components/internal/Documentation/PropsTable/compon
+    <li><span class="code">resolutions</span>Record&lt;String,{'{ url: string; width: number; height: number; }'} &gt;  
+   All available resolutions of your image that will be combined into a srcset statement</li>
+    <li><span class="code">sizes</span>string - Specfiy which size should be used with which viewport</li>
+    <li><span class="code">lazy</span>boolean - Specify if the image should be loaded only if it is visible in the viewport</li>
+    <li><span class="code">border</span>boolean - Specify if the shift / border should be applied</li>
+    <li><span class="code">zoom</span>boolean - Specify if the zoom-effect should be applied</li>
+    <li><span class="code">opacity</span>string - Specify if a black overlay with specified opacity should be applied on top. Possible values are "0" | "25" | "40" | "50" | "75" | "80"</li>
     </ul>
   </PropsTableRow>
-  <PropsTableRow value="tags" type="Array">
-      Specify which tags should be displayed. Required attributes are:
-      <ul>
-        <li><span class="code">label</span> Label of the tag</li>
-      </ul>
-
-  </PropsTableRow>
   <PropsTableRow value="teaser" type="String">
-    Specify which teaser should be displayed above the picture.
+    Text between headline and picture.
   </PropsTableRow>
   <PropsTableRow value="date" type="String">
-      Specify which date should be displayed.
+      Date when the article was written
+  </PropsTableRow>
+  <PropsTableRow value="tags" type="Array">
+      Tags under the article. Possible attributes are:
+      <ul>
+        <li><span class="code">label</span>string - Required - Label of the tag</li>
+        <li><span class="code">id</span>number | string -  Id of the tag</li>
+      </ul>
   </PropsTableRow>
   <PropsTableRow value="author" type="String">
-      Specify which author should be displayed.
+    Author of the article
   </PropsTableRow>
   <PropsTableRow value="handleTagClick" type="Function">
       You can pass in an optional callback that will be triggered, when a tag
       is clicked.
     </PropsTableRow>
+  <PropsTableRow value="returnText" type="String">
+    The text that will be displayed on the return button under the article.
+  </PropsTableRow>
   <PropsTableRow value="handleReturnClick" type="Function">
         You can pass in an optional callback that will be triggered, when the return button
             is clicked.
   </PropsTableRow>
-  <PropsTableRow value="socialText" type="String">
-    The text that will be displayed above the social media icons in the footer.
-  </PropsTableRow>
-  <PropsTableRow value="returnText" type="String">
-    The text that will be displayed on the return button in the footer.
+  <PropsTableRow value="social" type="Object">
+    Information about sociallinks. Possible attributes are:
+    <ul>
+          <li><span class="code">title</span>string - Title of the social area</li>
+          <li><span class="code">items</span>Object - social media items. Possible attributes are
```

Comment:
There is something wrong with the rendering of the nested lists on chrome. Please have another go at it.